js/ui/grid_core/ui.grid_core.adaptivity.js

```diff
@@ -1107,6 +1133,8 @@ export default {
                         changeType: 'update',
                         rowIndices: [oldExpandLoadedRowIndex - rowIndexDelta, newExpandLoadedRowIndex - rowIndexDelta]
                     });
+
+                    this.getController('adaptiveColumns').toggleCommandAdaptiveAriaLabel(key, oldKey);
```

Comment:
Call toggleCommandAdaptiveAriaLabel two times for key and oldKey because old adaptive row is hidden after expand another new adaptive row.